test/models/erratum_test.rb

```diff
@@ -90,7 +98,7 @@ def test_not_applicable_to_hosts
     def test_update_from_json
       errata = katello_errata(:security)
       json = errata.attributes.merge('description' => 'an update', 'updated' => Time.now, 'reboot_suggested' => true)
-      errata.update_from_json(json)
+      errata.update_from_json(errata_json(json))
```

Comment:
i think just calling  `errata.update_from_json(json.as_json)`
would work here.  It seemed to work fine for me locally.